### DISCUSSION TITLE
Problem with Mac: BigSur

### DISCUSSION THREAD

**Discussion by vimalmistry** [uNetworking/uWebSockets.js#563] [General]
I just upgraded my Mac to BigSur. And I am getting this errors.

```
{project}/node_modules/uWebSockets.js/uws.js:30
                throw new Error('This version of µWS is not compatible with your Node.js build:\n\n' + e.toString());
                      ^
Error: This version of µWS is not compatible with your Node.js build:

Error: Cannot find module './uws_darwin_x64_93.node'
```

I tried reinstall.

**Comment by e3dio** [comment]
You are not running the latest version, can tell by your error line number `uws.js:30`

**Reply by ghost** [reply]
I bet 5 USD it is because of NPM failures

**Reply by vimalmistry** [reply]
![Screenshot 2021-06-22 at 8 37 17 PM](https://user-images.githubusercontent.com/3993163/123008449-84d6d300-d3d8-11eb-9420-828727716129.png)

**Reply by e3dio** [reply]
Latest version is [19.2](https://github.com/uNetworking/uWebSockets.js/releases)
